sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/src/main/resources/META-INF/spring.provides

```diff
@@ -0,0 +1 @@
+provides: spring-cloud-azure-appconfiguration-config
```

Comment:
This file can be deleted. Refs: https://github.com/spring-projects/spring-boot/issues/1926